rules/scala/private/core.bzl

```diff
@@ -395,6 +395,23 @@ def _resource_path(file, strip_prefix):
             return rest
     return file.short_path
 
+def _build_deployable(ctx, jars_list):
+    # This calls bazels singlejar utility.
+    # For a full list of available command line options see:
+    # https://github.com/bazelbuild/bazel/blob/master/src/java_tools/singlejar/java/com/google/devtools/build/singlejar/SingleJar.java#L311
+    args = ["--normalize", "--sources"]
+    args.extend([j.path for j in jars_list])
```

Comment:
Use Bazel's args.add_all instead.

It prevents having to always expand the entire transitive list at analysis time.